### PR TITLE
Add travis build

### PR DESCRIPTION
**Pull request by streeve**
Add Travis CI, with plans for testing

### CODE REVIEW

**Comment by streeve** [comment]
I'm working on the tests now and can make this WIP if you think it should all be added in one PR

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+    make -j2 &&
+    make install &&
+    popd &&
+    popd
+  - pwd
+  - git clone --depth=1 https://github.com/ECP-copa/Cabana.git
+  - pushd Cabana && mkdir build && pushd build &&
+    cmake -DCMAKE_PREFIX_PATH=$HOME/kokkos
+          -DCMAKE_INSTALL_PREFIX=$HOME/Cabana
+          -DCabana_ENABLE_Serial=ON -DDCabana_ENABLE_OpenMP=ON
```
Comment (sslattery) [inline .travis.yml]: `-DDCabana_ENABLE_OpenMP` has an extra `D`

**Review by sslattery — APPROVED** [review]

File: .travis.yml (hunk 1 above)
Comment (streeve) [reply .travis.yml]: @rfbird since I stole this from CabanaPIC, that's hiding there as well

Comment (rfbird) [reply .travis.yml]: Thanks for pointing this out. I thought I lifted this from the Cabana repo, but It looks fine over there.

**Pull request merged**